packages/harfbuzz/build.sh

```diff
@@ -26,7 +27,7 @@ termux_step_post_get_source() {
 }
 
 termux_step_pre_configure() {
-	termux_setup_gir
+	TERMUX_PKG_VERSION=. termux_setup_gir
```

Comment:
I'm not sure I understand what's going on here.
Are we sourcing `termux_setup_gir` to get the version?

Reply:
By setting `TERMUX_PKG_VERSION=.` variable, termux_setup_gir function will look into gir directory instead of `gir/x.y.z`. This idea is used in other packages also.

Reply:
Apparently so, had never seen that before but yep, you're right.
I also see what you're doing now.
`TERMUX_PKG_VERSION=.` is being set *only* for the call to `termux_setup_gir`